app/controllers/ontologies_controller.rb

```diff
@@ -147,6 +147,7 @@ def edit
   def mappings
     @ontology_acronym = @ontology.acronym || params[:id]
     @mapping_counts = mapping_counts(@ontology_acronym)
+    @ontologies_mapping_count = LinkedData::Client::HTTP.get("#{MAPPINGS_URL}/statistics/ontologies")
```

Comment:
what is the difference between @mapping_counts and @ontologies_mapping_count

Reply:
@mapping_counts: the count of the mappings between each ontology and the current ontology.
@ontologies_mapping_count: the global mappings count for each ontology (used to draw the bubble view)